.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Build & Deploy to NPM
+
+on:
+  push:    
+    tags:        
+      - '*'
+    branches: [ main ]
```

Comment:
I see that this is set up to run every time there is a push to `main`, which is fine for testing purposes. 

However, in production means that these modules will be published to npm possibly _before_ they are published to deno.land, since commits `main` are not synced with the release schedule currently. 

Instead I recommend that we trigger this Github workflow every time that a [release](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release) event occurs. When Neil or Brad are updating this module, they will publish a release. You can see a list of previous releases [here](https://github.com/slackapi/deno-slack-sdk/releases)

You can also see from the event payload that that a GITHUB_REF value is provided, this you can use in line 33 instead of a hardcoded value (0.1.0) so that the version we publish to NPM also matches the version published to deno.land.